### ISSUE TITLE
Is it possible to give an option to set mCenterX and mCenterY to be the X and Y positions of the initial touch?

### ISSUE THREAD

**Issue by teancake** [controlwear/virtual-joystick-android#2]
This is a great library and I am now using it in my app to control a quadcopter's thrust and yaw angle. But gradually I found a problem. 

Each time I touch the joystick on the screen, I wish the angle and the strength to be 0. Despite very careful touches, the two values were scarcely 0 and my quadcopter was disturbed by this initial nonzero touch. 

If the mCenterX and mCenterY are defined at the initial touch position, then there is no need to touch the exact centre of the circle each time. Most importantly, it liberates the eyes from the touch screen and allows for blind touches and move. 

A responsive region may need to be defined accordingly, since we don't want the whole screen to respond to a single joystick, and there may be multiple joysticks on one screen.

I am not sure whether this make sense, but thanks in advance if you could kindly consider it.

**Comment by makowildcat** [comment]
Hello @teancake, 
Thank you for reporting this issue, indeed your comment is very relevant, I’ll work on it as soon as possible (probably this weekend or next week.
I’ll probably give the choice to have with a fixed Center (as it is right now) or an auto-defined Center related to initial touch as you mentioned.

**Comment by teancake** [comment]
Hi @makowildcat , thanks a lot. :D

**Comment by makowildcat** [comment]
Hello, @teancake,
It's done, just get the new version 1.0.0
I upload the documentation, the default behavior is FixedCenter and you can set up (either in xml or in java) a parameter to auto-defined center related to the initial touch as you explained.
Hope it's going well for your need, let me know if you see something wrong.
:)

**Comment by teancake** [comment]
Hi @makowildcat 

I tried your modified code. The auto-defined centre worked perfectly. But the behaviour of the joystick was a little bit funny :P -- the whole joystick, i.e. both the dot and the circle, moved to the initial touch position. 

What I was thinking is a bit different: the joystick UI remains at the prespecified position while the computation of the mPosX and mPosY uses the initial touch position if fixedCenter = false. I am not sure whether this feature is reasonable and whether my meaning is clear. I've created a pull request with all necessary modifications so that you know what I am saying. :D 

Thanks a lot.
